app/src/main/java/cmpe277/project/skibuddy/ParticipantAdapter.java

```diff
@@ -74,8 +75,12 @@ public View getView(int position, View convertView, ViewGroup parent) {
         // Change icon based on name
         String s = participant.getName();
 
-        System.out.println("events "+ s);
-        holder.logo.setImageResource(R.drawable.invite);
+        LoadProfilePicture profilePicture  = new LoadProfilePicture(holder.logo);
+        profilePicture.loadPicture(participant);
+
+        System.out.println("events " + s);
+        //holder.logo.setImageResource(R.drawable.invite);
```

Comment:
Could you please remove commented out code? Also, if this drawable isn't used anymore delete it